A32NX/html_ui/Pages/A32NX_Core/A32NX_Speeds.js

```diff
@@ -29,7 +29,7 @@ class A32NX_Speeds {
          * On disagree cache gets updated and Vspeeds recalculated, then shared.
          */
         setInterval(() => {
-            const fp = Simplane.getCurrentFlightPhase();
+            const fp = SimVar.GetSimVarValue("L:A32NX_FWC_FLIGHT_PHASE", "Enum");
```

Comment:
This should surely be the FMGC flight phase?

Reply:
Yes, it should be, I changed it.